app/src/main/java/com/example/fbufinalapp/MainActivity.java

```diff
@@ -38,6 +41,26 @@ protected void onCreate(Bundle savedInstanceState) {
         View view = binding.getRoot();
         setContentView(view);
 
+        if (ParseUser.getCurrentUser() == null){
```

Comment:
Need some clarification on how this part of the code works. 

It checks if no user is there. 

If no user is there, it creates a default "guest" user. 

1. Guest users cannot have their itineraries persist in state, correct? 
2. When/how often does this code run in the actual flow?

Reply:
Yes! So this line will check if there is a user current logged in. If there is no user, it'll create that guest account. 
1. At the moment, their itineraries do persist across states and when they sign up for a real account. Maybe this isn't how guest accounts usually work? 
2. The code runs once at the very beginning, before the rest of the app loads! I don't think it's run again until the app restarts